hs_core/hydroshare/utils.py

```diff
@@ -794,20 +794,29 @@ def prepare_resource_default_metadata(resource, metadata, res_title):
         metadata.append({'creator': creator_data})
 
 
-def get_party_data_from_user(user):
-    party_data = {}
+def get_user_party_name(user):
     user_profile = get_profile(user)
-
-    if user_profile.middle_name:
-        user_full_name = '%s, %s %s' % (user.last_name, user.first_name,
-                                        user_profile.middle_name)
+    if user.last_name and user.first_name:
+        if user_profile.middle_name:
```

Comment:
Stylistically, I've been seeing most people these days using "{}".format()

Reply:
Agreed for using format(), and did not make the stylistic change for the exact reason you mentioned. There are multiple places throughout our code base where this old formatting style is used, so I thought I don't have to make this stylistic change here as long as it works.

Reply:
We could start using f-strings now, too